website/content/en/blog/clusters-for-all-cloud-tenants.md

```diff
@@ -1,7 +1,12 @@
 ---
 title:  "Clusters for all cloud tenants"
 date:   2022-06-02 13:04:00 +0200
-author: Josh Gavant 
+author: Josh Gavant
+categories:
+- Community
```

Comment:
Perhaps - "Article" and put "Community-contributed" as a tag?

Reply:
I like this!